tests/wsl/firstrun.pm

```diff
@@ -165,17 +166,17 @@ sub run {
     # Nothing to do in WSL2 pts w/o serialdev support
     # https://github.com/microsoft/WSL/issues/4322
     if (get_var('WSL2')) {
-        enter_cmd "exit";
+        enter_cmd_slow "exit\n";
         return;
     }
 
     is_fake_scc_url_needed || become_root;
     assert_script_run 'cd ~';
     assert_script_run "zypper ps";
-    enter_cmd 'exit';
+    enter_cmd_slow "exit\n";
     sleep 3;
     save_screenshot;
-    is_fake_scc_url_needed || enter_cmd 'exit';
+    is_fake_scc_url_needed || enter_cmd_slow "exit\n";
```

Comment:
Isn't `enter_cmd_slow` always hitting the enter/newline?

Reply:
Not sure, so I added it just in case... :man_shrugging: